### PR TITLE
Migrate to python 3

### PR DESCRIPTION
**Pull request by khashf**
Python 2 EOLed in 2020: https://pythonclock.org/

### CODE REVIEW

**Comment by khashf** [comment]
[Python 2 -> Python 3 migration](https://trello.com/c/Z8tP7mjt/1127-python-2-python-3-migration)

**Comment by osuosl-manatee** [comment]
This PR has been automatically built and can be viewed here: http://wiki-44.staging.osuosl.org

File: source/conf.py [hunk 1]
```diff
@@ -45,8 +45,8 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'OSL Wiki'
-copyright = u'2014, Oregon State University'
+project = 'OSL Wiki'
+copyright = '2014, Oregon State University'
```
Comment (ramereth) [inline source/conf.py]: If you're doing this, you might as well update this to the following:
```suggestion
copyright = '2014-2019, Oregon State University'
```

**Comment by osuosl-manatee** [comment]
This PR has been automatically built and can be viewed here: http://wiki-44.staging.osuosl.org

**Pull request merged**